main.go

```diff
@@ -41,6 +42,16 @@ func main() {
 	case "untar":
 		checkArguments(args, 3)
 		err = untar(args[1], args[2])
+	case "date":
+		format := time.RFC3339
+		if len(args) == 1 {
+			format = args[0]
```

Comment:
In my opinion we could just have RFC3339 for now and remove the format argument: «just a stupid date» 📆

Reply:
Sounds fair 😛